modules/shared/src/main/scala/org/tessellation/shared/package.scala

```diff
@@ -68,29 +68,29 @@ package object shared {
     classOf[GlobalSnapshot] -> 600,
     classOf[StateChannelSnapshotBinary] -> 601,
     classOf[SnapshotOrdinal] -> 602,
-    classOf[DAGBlock] -> 603,
-    DAGBlock.OrderingInstance.getClass -> 604,
+    classOf[Block] -> 603,
+    Block.OrderingInstance.getClass -> 604,
     classOf[BlockReference] -> 605,
     classOf[GlobalSnapshotInfoV1] -> 606,
     classOf[SnapshotTips] -> 607,
     classOf[ActiveTip] -> 608,
     ActiveTip.OrderingInstance.getClass -> 609,
-    classOf[BlockAsActiveTip[_]] -> 610,
-    DAGBlock.OrderingInstanceAsActiveTip.getClass -> 611,
+    classOf[BlockAsActiveTip] -> 610,
+    BlockAsActiveTip.OrderingInstance.getClass -> 611,
     classOf[DeprecatedTip] -> 612,
     DeprecatedTip.OrderingInstance.getClass -> 613,
     classOf[Tips] -> 614,
     classOf[GlobalIncrementalSnapshot] -> 615,
     classOf[MerkleRoot] -> 616,
     classOf[GlobalSnapshotInfo] -> 617,
-    classOf[CurrencyTransaction] -> 618,
-    classOf[CurrencyBlock] -> 619,
+    // classOf[Transaction] -> 618,
```

Comment:
I guess CurrencyTransaction should remain here as a commented out value

Reply:
:+1: